packages/types/src/Bytes.ts

```diff
@@ -19,6 +20,14 @@ export default class Bytes extends U8a {
     return this.length + Compact.encode(this.length).length;
   }
 
+  toHex (): string {
```

Comment:
Not sure if you need it in the UI, but in #172 we could add toHex/fromHex to all types?

Reply:
That would actually be nice - it currently is only on the U8a ones, but that probably will make sense to all for the sake of consitency.